src/App.tsx

```diff
@@ -1,11 +1,57 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useRef, useState,
+} from 'react';
+import classNames from 'classnames';
 import { AuthContext } from './components/Auth/AuthContext';
+import { TodoList } from './components/Auth/TodoList';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { ErrorNotification } from './components/Auth/ErrorNotification';
+import { SortType } from './types/filterBy';
+
+function filterTodos(
+  todos: Todo[],
+  sortType: SortType,
+) {
+  const visibleTodos = [...todos];
+
+  switch (sortType) {
+    case SortType.Active:
+      return visibleTodos.filter(todo => !todo.completed);
+
+    case SortType.Completed:
+      return visibleTodos.filter(todo => todo.completed);
+    default:
+      return visibleTodos;
+  }
+}
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [sortType, setSortType] = useState<SortType>(SortType.All);
+  const [completeItem, setCompleteItem] = useState<number>(0);
+  const [selectedLink, setSelectedLink] = useState<string>('All');
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  useEffect(() => {
+    getTodos(5)
```

Comment:
don't hardcode user ID for getTodos

Reply:
It just for easy understanding in this part of task, in the next tasks I changed it